src-tauri/src/lib.rs

```diff
@@ -294,9 +297,28 @@ pub fn launch_northstar(
     }
 
     // Only Windows with Steam or Origin are supported at the moment
-    if host_os == "windows"
-        && (matches!(game_install.install_type, InstallType::STEAM)
-            || matches!(game_install.install_type, InstallType::ORIGIN)
+    if matches!(game_install.install_type, InstallType::STEAM)
+    {
+        // Create run_northstar.txt for NorthstarProton
+        if host_os != "windows"
+        {
+            let mut file = std::fs::OpenOptions::new()
+                .create(true)
+                .write(true)
+                .truncate(true)
+                .open("run_northstar.txt").unwrap();
+
+            let mut content: &[u8] = b"1";
+            std::io::copy(&mut content, &mut file).unwrap();
+        }
+
+        return match open::that("steam://run/1237970/") {
```

Comment:
Another thing that needs to be considered is NorthstarProton on Linux.

Steam only parses available Proton versions on startup, and will ignore modification to `config.vdf`.

This means we need Steam closed and then restarted (or at least instruct the user on how to set it up)

Reply:
Hmm, I really wish there was a way to tell Steam to use a specific Proton version for a single run of a game. Killing and restarting Steam on every launch really doesn't sound like a good idea ^^

Reply:
I don't think we can do that either way, we cannot control how the user set up Steam.

Best we can do is setup NorthstarProton (libthermite provides the capbilities, we just need to do it) and then check if NorthstarProton is the compatibility tool setup for Titanfall.

The user would only need to be instructed to restart Steam and change Proton and be informed when Titanfall isn't setup to use the right Proton.